Source/Libraries/Vanguard/Vanguard.csproj

```diff
@@ -89,4 +89,4 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-</Project>
+</Project>
```

Comment:
Any idea why this happened? I know there's no technical reason for csproj itself to have newline at end of file and the standard is moreover a leftover from c. I think the editorconfig might be affecting this setting because it seems like depending on which project I save in, sometimes it'll delete the newline.

Reply:
I have my VS Code configured to automatically _add_ EOF newlines. I recently switched to using Visual Studio, which for some reason, is configured to remove these EOF newlines from the csproj files. I think that's what's happening

Reply:
That could be it. Sometimes I'll make edits in vs code as it's faster than waiting for VS to launch so maybe that's why it's popping back and forth. I could have sworn I had editorconfig override csproj but it may only acknowledge that if you do it with the text editor and not edit something in the IDE itself.

Reply:
Related issue here: https://github.com/dotnet/project-system/issues/4993

Reply:
For the purposes of this PR I removed the .csproj changes, but I know that this will be like playing whack-a-mole in the long term if we don't decide one way or the other.